site/app/views/PDFView.php

```diff
@@ -38,7 +38,9 @@ public function showPDFEmbedded($params) {
         $localjs[] = $this->core->getOutput()->timestampResource(FileUtils::joinPaths('pdfjs', 'pdf.worker.min.js'), 'vendor');
         $localjs[] = $this->core->getOutput()->timestampResource(FileUtils::joinPaths('pdf-annotate.js', 'pdf-annotate.min.js'), 'vendor');
         $localjs[] = $this->core->getOutput()->timestampResource(FileUtils::joinPaths('pdf', 'PDFAnnotateEmbedded.js'), 'js');
-
+        $localjs[] = $this->core->getOutput()->timestampResource(FileUtils::joinPaths('html2canvas', 'html2canvas.js'), 'vendor');
```

Comment:
you can use 
```php
$this->core->getOutput()->addVendorJs(FileUtils::joinPaths('html2canvas', 'html2canvas.js'));

```

We shouldn't be calling the timestampResource directly here but that can be handled when pdf annotation is re-written